liberty-maven-plugin/src/main/resources/net/wasdev/wlp/maven/plugins/MvnMessages.properties

```diff
@@ -341,3 +341,5 @@ warn.invalid.app.directory.useraction=Change the appsDirectory configuration par
 info.default.app.directory=CWWKM2185I: The liberty-maven-plugin configuration parameter "appsDirectory" value defaults to "{0}".
 info.default.app.directory.explanation=The default value for the "appsDirectory" configuration parameter is set to "apps" if the application is defined in the server configuration, otherwise it is set to "dropins".
 info.default.app.directory.useraction=No action is required.
+
+�=The Liberty {0} Ant task was not found. Verify that the required version of the wlp-anttasks.jar is available.
```

Comment:
something is wrong with this message.  It should have `error.dependencies.not.found` to the left of the equals sign.  There should also be entries for error.dependencies.not.found.explanation and error.dependencies.not.found.useraction following the pattern above.

error.dependencies.not.found.explanation=The required version of the Liberty Ant tasks is not available in the local maven repository.
error.dependencies.not.found.useraction=Verify that you can connect to the Maven repository to download the latest dependencies